models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  has_secure_password
+  validates_format_of :email, :with => /\A[^@\s]+@([^@\s]+\.)+[^@\s]+\z/
+  validates_uniqueness_of :username
+  validates_uniqueness_of :email
+  has_many :posts, :dependent => :destroy
```

Comment:
Just our of curiosity, does it mean if the user deletes its account all their peeps will be destroyed? It might make sense not to do it and keep them in a database...